Formula/n/newnode-helper.rb

```diff
@@ -0,0 +1,39 @@
+class NewnodeHelper < Formula
+  desc "Web proxy that uses a distributed p2p network to circumvent censorship"
+  homepage "https://www.newnode.com/newnode-vpn"
+  url "https://github.com/clostra/newnode.git",
+    tag: "2.1.4", revision: "c42a04ded55cfdff857878f4d9234950e530bb00"
+  license "GPL-2.0-only"
+
+  depends_on xcode: ["9.3", :build]
+  depends_on "automake"
+  depends_on "coreutils"
+  depends_on "libtool"
+  depends_on "mbedtls@2"
```

Comment:
Does it work with the modern version? This is only supported for a couple of months.

Reply:
It doesn't right now. We can add this to upstream later as a dependency and update the formula.

Reply:
I'm hesitant to accept a new formula that depends on something that will no longer be supported later this year. When `mbedtls@2` loses support, we will deprecate anything still using it that can't be updated. In our experience, promises of "let's get this added and we'll fix it later" very rarely work out.

Reply:
The new version 2.1.5 should not have any need for mbedtls.